lldp/lldp.yaml

```diff
@@ -221,3 +230,92 @@ components:
+          x-field-uid: 2
+        information:
+          description: >-
+            Contains the organizationally defined information. The actual format of the organizationally defined information string 
+            field is organizationally specific and can contain either binary or alpha-numeric information that is instance specific 
+            for the particular TLV type and subtype. Alpha-numeric information are encoded in UTF-8 (as specified in IETF RFC 3629). 
+            Or include one or more information fields with their associated field-type identifiers, designators similar to those in 
+            the Management Address TLV.
+          $ref: '#/components/schemas/Lldp.OrgInfoType'  
+          x-field-uid: 3
+
+    Lldp.OrgInfoType:
+      description: >-
+        Contains either the Alpha-numeric information encoded in UTF-8 (as specified in IETF RFC 3629) or include one or more information fields with 
+        their associated field-type identifiers designators, similar to those in the Management Address TLV.
+      type: object
+      properties:
+        choice:
+          description: >-
+            In info mode the organizationally defined information contain either binary or alpha-numeric information encoded in UTF-8 
+            (as specified in IETF RFC 3629). Subinfo mode will contain a array of sub tlvs.
+          type: string
+          default: info 
+          x-field-uid: 1
+          x-enum:
+            info:
+              x-field-uid: 1
+            subinfo:
+              x-field-uid: 2
+        info:
+          description: >-
+            The organizationally defined information encoded in UTF-8 (as specified in IETF RFC 3629).
+          type: string
+          x-field-uid: 2
+        subinfos:
+          description: >-
+            The information field of Organization Specific TLV contains a array of sub tlvs.
+          type: array
+          items:
+            $ref: '#/components/schemas/Lldp.OrgSubInfo' 
+          x-field-uid: 3
+
+    Lldp.OrgSubInfo:
+      description: >-
+        The sub tlv contains of information field of Organization Specific TLV. 
+      type: object
+      required: [info]
+      properties:
+        type:
+          description: >-
+            The organizationally defined type field shall contain a unique subtype value assigned by the defining organization.
+          type: integer
+          format: uint32
+          minimum: 1
+          maximum: 127
+          default: 1
+          x-field-uid: 1
+        info:
+          description: >-
+            Contains the organizationally defined information. The actual format of the organizationally defined information string 
+            field is organizationally specific and can contain either binary or alpha-numeric information that is instance specific 
+            for the particular TLV type and subtype. Alpha-numeric information are encoded in UTF-8 (as specified in IETF RFC 3629). 
+          type: string
```

Comment:
If type is string how can we send binary data in it.

Reply:
updated